js/script.js

```diff
@@ -0,0 +1,85 @@
+window.onload = function() {
+  const mainInput = document.getElementById("new-todo-title");
+  const mainList = document.getElementById("todo-list");
+  const listCount = document.querySelector(".todo-count strong");
+
+  mainInput.addEventListener("keydown", e => {
+    // console.log(e.key);
+    if(e.key === "Enter" && mainInput.value.replace(/ /g,"") != "") {
```

Comment:
```suggestion
    if(e.key === "Enter" && mainInput.value.trim() !== "") {
```
이렇게 `trim()`을 사용하여 표현할 수 있습니다!

Reply:
감사합니다!